lib/zstd.h

```diff
@@ -1297,16 +1297,29 @@ ZSTDLIB_API unsigned long long ZSTD_decompressBound(const void* src, size_t srcS
  *           or an error code (if srcSize is too small) */
 ZSTDLIB_API size_t ZSTD_frameHeaderSize(const void* src, size_t srcSize);
 
+typedef enum {
+  ZSTD_sf_explicitBlockDelimiters,  /* Representation of ZSTD_Sequence contains explicit block delimiters */
+  ZSTD_sf_noBlockDelimiters         /* Representation of ZSTD_Sequence has no block delimiters, sequences only */
+} ZSTD_sequenceFormat_e;
+
```

Comment:
I wonder if it makes sense to create another struct like
```struct ZSTD_Sequences {
    ZSTD_Sequence* seqs,
    ZSTD_sequenceFormat_e format
}```

In order to force a tighter coupling of the concept of the format and the `ZSTD_Sequence` array. My main hesitation is that I don't want to create too many new types for this API.

Reply:
That doesn't seem necessary